test/__snapshots__/orders.test.ts.snap

```diff
@@ -2,7 +2,7 @@
 
 exports[`Orders > should cancel an order 1`] = `
 {
-  "cancellation_message": "Cancelled by thomas_kambio_app during picking up",
+  "cancellation_message": Any<String>,
```

Comment:
I tossed around whether to use [`expect.stringMatching`](https://vitest.dev/api/expect.html#expect-stringmatching) here, but it's safer to just ensure that this exists and is a string than to confirm the format – the wording of the message might change in future.